sktime/transformations/compose/_ytox.py

```diff
@@ -6,6 +6,8 @@
 __all__ = ["YtoX"]
 
 from sktime.transformations.base import BaseTransformer
+from sktime.transformations.series.boxcox import BoxCoxTransformer
```

Comment:
I would suggest to shift these imports inside the get_test_params method instead of importing them at the top level